src/ListActions/ListActions.js

```diff
@@ -58,202 +45,87 @@ var ListActionActivationSupport = {
+* {@link module:moonstone/ListActions~ListActionsPopup} is a
+* [control]{@link module:moonstone/ContextualPopup~ContextualPopup} used by
 * {@link module:moonstone/ListActions~ListActions} to house a menu of selectable options.
 *
-* @class ListActionsDrawer
-* @extends module:enyo/Control~Control
+* @class ListActionsPopup
+* @extends module:moonstone/ContextualPopup~ContextualPopup
 * @ui
 * @public
 */
-var ListActionsDrawer = kind(
-	/** @lends module:moonstone/ListActions~ListActionsDrawer */ {
-
-	/**
-	* @private
-	*/
-	name: 'moon.ListActionsDrawer',
-
-	/**
-	* @private
-	*/
-	kind: Control,
-
-	/**
-	* @private
-	* @lends module:moonstone/ListActions~ListActionsDrawer.prototype
-	*/
-	published: {
-
-		/**
-		* If `true`, the drawer will be in its opened state; otherwise, it will be closed.
-		*
-		* @type {Boolean}
-		* @default false
-		* @public
-		*/
-		open: false
-	},
-
-	/**
-	* @private
-	*/
-	handlers: {
-		onPanelOnscreen: 'hideCloseButton'  // Re-evaluate whether ListActions should be open and if the close X should show or not.
-	},
-
-	/**
-	* @private
-	*/
-	classes: 'moon-list-actions-drawer',
+var ListActionsPopup = ContextualPopup.kind(
+	/** @lends module:moonstone/ListActions~ListActionsPopup */ {
 
 	/**
 	* @private
 	*/
-	components: [
-		{name: 'client', kind: Control, classes: 'moon-list-actions-drawer-client moon-neutral', ontransitionend: 'handleTransitionEnd'}
-	],
+	classes: 'moon-list-actions-popup',
 
 	/**
-	* @private
-	*/
-	events: {
-		onComplete: '',
-		onCustomizeCloseButton: ''
-	},
-
-	/**
-	* @fires module:moonstone/ListActions~ListActionsDrawer#onComplete
-	* @private
-	*/
-	rendered: function () {
-		Control.prototype.rendered.apply(this, arguments);
-		// Temporarily disable animation
-		this.applyAnimatedMode(false);
-		// Set the state of the drawer
-		this.openChanged();
-		// Re-enable animation
-		this.applyAnimatedMode(true);
-		// Let any watchers know we've finished our preparation
-		this.doComplete({rendered: true});
-	},
-
-	/**
-	* @fires module:moonstone/ListActions~ListActionsDrawer#onComplete
-	* @private
-	*/
-	handleTransitionEnd: function (sender, e) {
-		if (e.originator === this.$.client) {
-			this.doComplete();
-			return true;
-		}
-	},
-
-	/**
-	* We override `getBubbleTarget()` here so that events emanating from a
-	* [ListActionsDrawer]{@link module:moonstone/ListActions~ListActionsDrawer} instance will bubble to the owner
-	* of the associated [ListActions]{@link module:moonstone/ListActions~ListActions} instance, as expected. This
-	* is necessary because events normally bubble to a control's DOM parent, but we have
-	* sneakily arranged for the DOM parent of a `ListActionsDrawer` instance to be not the
-	* `ListActions` instance, but the containing [Header]{@link module:moonstone/Header~Header} instance.
+	* If `true`, {@glossary Spotlight} (focus) cannot leave the area of the popup unless the
```

Comment:
Instead of copying this description from `ContextualPopup`, we can replace this with

```
@see enyo/ContextualPopup~ContextualPopup#spotlightModal
```